### PR TITLE
Allow disable-element extension to disable multiple elements

### PR DESCRIPTION
**Pull request by imaperson1060**
Allow multiple elements to be selected in the query

Better version of #1639 - this time I actually read the contribution guide

### CODE REVIEW

**Comment by imaperson1060** [comment]
For some reason, when I run `npm test`, the hx-boost attribute fails. It's not like this in the browser and I didn't change anything related to hx-boost.

image
image

File: test/ext/disable-element.js [hunk 1]
```diff
@@ -59,4 +59,37 @@ describe("disable-element extension", function() {
         btn2.disabled.should.equal(false);
     });
 
+    it('disables multiple elements during htmx request', function () {
+		// GIVEN:
+		// - A button triggering an htmx request with disable-element extension
+		// - Another button that needs to be disabled during the htmx request
+		// - A control button that won't be disabled during the htmx request
+		// - All buttons are enabled
```
Comment (alexpetros) [inline test/ext/disable-element.js:67]: I appreciate the comments but I think the test is self-explanatory and it would be better to remove them :)

**Review by alexpetros — COMMENTED** [review]
Great PR; strict, intuitive enhancement and I can't think of any reason why this would cause a regression.

File: test/ext/disable-element.js (hunk 1 above)
Comment (imaperson1060) [reply test/ext/disable-element.js]: da89d61

File: src/ext/disable-element.js [hunk 2]
```diff
@@ -5,12 +5,14 @@ htmx.defineExtension('disable-element', {
...
         let elt = evt.detail.elt;
         let target = elt.getAttribute("hx-disable-element");
-        let targetElement = (target == "self") ? elt : document.querySelector(target);
+        let targetElements = (target == "self") ? [ elt ] : document.querySelectorAll(target);
 
-        if (name === "htmx:beforeRequest" && targetElement) {
-            targetElement.disabled = true;
-        } else if (name == "htmx:afterRequest" && targetElement) {
-            targetElement.disabled = false;
+		for (var i = 0; i < targetElements.length; i++) {
```
Comment (1cg) [inline src/ext/disable-element.js:10]: indentation looks a little funky here.  tabs?

**Review by 1cg — CHANGES_REQUESTED** [review]

File: src/ext/disable-element.js (hunk 2 above)
Comment (imaperson1060) [reply src/ext/disable-element.js:10]: Yeah, I noticed that in the GitHub editor. The other lines use spaces but I have my IDE set to use tabs instead. In most places the tab size is 4 spaces so it's not noticeable, but for some reason the GitHub editor has like 8.

Comment (alexpetros) [reply src/ext/disable-element.js:10]: Can you re-indent to match the spaces scheme of the project please?

**Pull request merged**